source/transfer_list.rst

```diff
@@ -394,26 +370,28 @@ ends at offset *hdr_size + data_size*, from the start of the entry.
      - Description
 
    * - tag_id
-     - 0x4
+     - 0x3
      - 0x0
      - The tag_id field must be set to **4**.
 
    * - hdr_size
-     - 0x4
-     - 0x4
-     - The size of this entry header in bytes.
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
 
    * - data_size
      - 0x4
-     - 0x8
-     - The size of the data content in bytes.
+     - 0x4
+     - The size of all included ACPI tables + padding in bytes.
 
    * - reserved
-     - 0x4
-     - 0xc
+     - align16(hdr_size) - hdr_size
+     - hdr_size
```

Comment:
I've just noticed this item and the one after don't render well in the pdf

Reply:
This table just lists sizes and offsets, not full addresses, so none of it should depend on te_base_addr. Maybe you meant `align16(te_base_addr + hdr_size) - (te_base_addr + hdr_size)`? That's correct but that's just a more complicated version of what I wrote, since we already know te_base_addr is 16-byte aligned, so `align16(te_base_addr + hdr_size) = te_base_addr + align16(hdr_size)`.

(I'll try to figure out how to get the PDF building to work. Is there anything obvious I can do to make it look better without being able to see it?)

Reply:
I agree that if the te_base_addr is 16-byte aligned, what you have already works, but in another comment I was suggesting removing that alignment requirement, since this could get onerous with larger alignment sizes. In that case I think the size of the reserved field would depend on the te_base_addr. Your alignment calculation is correct, though.

Sorry, I can't suggest a quick fix to make this render better. The CI emits a rendered version in case that helps (e.g. https://github.com/FirmwareHandoff/firmware_handoff/actions/runs/3715576748)